raiden/tests/unit/test_accounts.py

```diff
@@ -80,7 +80,7 @@ def test_account_manager_invalid_files(test_keystore, caplog):
         AccountManager(test_keystore)
 
     for msg, file_name, reason in [
-        ('Invalid account file', KEYFILE_INVALID, 'No JSON object'),
+        ('Invalid account file', KEYFILE_INVALID, 'Expecting value: line 1 column 1 (char 0)'),
```

Comment:
It's better to fix the AccountManager to inform a better error.

Reply:
See #1195